OpenUtau.Core/Classic/UstNote.cs

```diff
@@ -38,7 +38,8 @@ class UstNote {
                 modulation = (int)phoneme.GetExpression(project, track, Ustx.MOD).Item1;
                 flags = FlagsToString(phoneme.GetResamplerFlags(project, track));
                 if (phoneme.oto != null) {
-                    filename = phoneme.oto.DisplayFile;
+                    var relativePath = phoneme.oto.File.Replace(track.Singer.Location + "\\", "");
```

Comment:
You are hard-coding the windows path separator "\\". Does this work on platforms other than windows?